crates/ruff_python_parser/tests/snapshots/invalid_syntax@type_param_default_py312.py.snap

```diff
@@ -0,0 +1,199 @@
+                            },
+                        ),
+                    ],
+                },
+            ),
+            ClassDef(
+                StmtClassDef {
+                    range: 88..111,
+                    decorator_list: [],
+                    name: Identifier {
+                        id: Name("C"),
+                        range: 94..95,
+                    },
+                    type_params: Some(
+                        TypeParams {
+                            range: 95..104,
+                            type_params: [
+                                TypeVar(
+                                    TypeParamTypeVar {
+                                        range: 96..103,
+                                        name: Identifier {
+                                            id: Name("T"),
+                                            range: 96..97,
+                                        },
+                                        bound: None,
+                                        default: Some(
+                                            Name(
+                                                ExprName {
+                                                    range: 100..103,
+                                                    id: Name("int"),
+                                                    ctx: Load,
+                                                },
+                                            ),
+                                        ),
+                                    },
+                                ),
+                            ],
+                        },
+                    ),
+                    arguments: Some(
+                        Arguments {
+                            range: 104..106,
+                            args: [],
+                            keywords: [],
+                        },
+                    ),
+                    body: [
+                        Expr(
+                            StmtExpr {
+                                range: 108..111,
+                                value: EllipsisLiteral(
+                                    ExprEllipsisLiteral {
+                                        range: 108..111,
+                                    },
+                                ),
+                            },
+                        ),
+                    ],
+                },
+            ),
+        ],
+    },
+)
+```
+## Unsupported Syntax Errors
+
+  |
+1 | # parse_options: {"target-version": "3.12"}
+2 | type X[T = int] = int
+  |        ^^^^^^^ Syntax Error: Cannot use type parameter default on Python 3.12 (syntax was added in Python 3.13)
+3 | def f[T = int](): ...
+4 | class C[T = int](): ...
+  |
+
+
+  |
+1 | # parse_options: {"target-version": "3.12"}
+2 | type X[T = int] = int
+3 | def f[T = int](): ...
+  |       ^^^^^^^ Syntax Error: Cannot use type parameter default on Python 3.12 (syntax was added in Python 3.13)
```

Comment:
What about something like "Cannot set default type for a type parameter on Python 3.12 (...)" ?